### DISCUSSION TITLE
[PSD11] Exercise 3.1

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#171] [Esercizi (A-L)]
![image](https://user-images.githubusercontent.com/69389461/210061705-e51a0be4-18bc-4e3e-bd8e-1943cc11ff9d.png)

**Comment by Alesavio** [comment]
Ricordando che, in un S-R Latch, quando sia S che R valgono 1 si ha uno stato illegale, l'output Q dovrebbe risultare come in foto.

![0EDC591E-A706-4E7E-95AB-C502A2406466](https://user-images.githubusercontent.com/121877241/210374490-5ee50135-c866-45a0-b072-ed67a38b290b.jpg)

**Comment by fraxb04** [comment]
@Alesavio 
tutto corretto, ma non capisco dal disegno se dopo l'ultima illegalità Q sale nello stesso punto della fine della discesa di R oppure se sale all'inizio della discesa di R

**Comment by FeddyLix17** [comment]
@fraxb04 teoricamente sale pochissimo dopo la discesa di R
